exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/fn/JsonReader.java

```diff
@@ -402,7 +402,8 @@ private void writeDataAllText(MapWriter map, FieldSelection selection, boolean m
         handleString(parser, map, fieldName);
         break;
       case VALUE_NULL:
-        // do nothing as we don't have a type.
```

Comment:
Good point...these would fail to read due to the complex type field under all_text_mode.  Let me see if there's a common solution to both types of use cases (the ones in my unit tests and the ones here)..